### PR TITLE
[api] basic multibuild support

### PR DESCRIPTION
**Pull request by adrianschroeter**
Support some routes for multibuild packages:
- list built binaries
- get logfile
- get binary
- get buildinfo (for local build)

### CODE REVIEW

**Comment by adrianschroeter** [comment]
@mlschroe the backend delivers HTTP 200 when listing not exiting multibuild package build directories... seems to be a bug?

File: src/api/app/models/package.rb [hunk 1]
```diff
@@ -151,13 +151,23 @@ def self.internal_get_project(project)
     prj
   end
 
+  def self.normalize_package_name(name)
+    # exception for package names used to have a collon
```
Comment (hennevogel) [inline src/api/app/models/package.rb]: What are you trying to do here? There are packages in that have a colon in the name? How can this be?

Comment (adrianschroeter) [inline src/api/app/models/package.rb]: Because we have them? product containers did always contain a colon and this is according to our verification code since SLE 11 code ....

Comment (hennevogel) [inline src/api/app/models/package.rb]: But product 'containers' are covered by the first line in this method right?

File: src/api/test/test_helper.rb [hunk 2]
```diff
@@ -89,7 +89,8 @@ def backend_data
 def inject_build_job(project, package, repo, arch, extrabinary = nil)
   job=IO.popen("find #{backend_data}/jobs/#{arch}/ -name #{project}::#{repo}::#{package}-*")
   jobfile=job.readlines.first
-  return unless jobfile
+  return if project == "BrokenPublishing"
```
Comment (hennevogel) [inline src/api/test/test_helper.rb]: Why is this needed now?

Comment (adrianschroeter) [inline src/api/test/test_helper.rb]: it was also needed before, but the new way ignores the error only for this single case and we get an error now when it happens in some other situation. Before a missing jobfile was always ignored.
